support-lambdas/stripe-intent/README.md

```diff
@@ -0,0 +1,14 @@
+# Stripe Intent lambda plus API gateway
+
+This lambda just waits for a request, then loads the support-worker
+config for stripe, and gets a setup intent id for the relevant
+stripe account.
+
+It is a lambda because it doesn't need a whole auto scaling group,
+the latency is not critical, and doing it this way provides better
+separation between codebases.
+
+It is a separate jar from the support-workers jar because the unused
+code would add to be loaded on every single step of the functions.
+Hopefully we can be a bit more cohesive and minimise dependencies if
+we separate things better.
```

Comment:
this turned out to be harder than expected, for example because pulling in support-models pulls in acquisition-event-producer which pulls in various thrift stuff including a 3.6mb binaries of compression algorithm used by capi.  I think sometimes pulling in a "com.gu" library can get more than we bargained for as we are not always disciplined at not adding dependencies that not everyone would need.  Client side devs have some nice tooling so every import shows the size in KB of the file in the IDE, but server side we have no idea.